<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiSub1xDialog</class>
 <widget class="QDialog" name="MultiSub1xDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultiSub1x</string>
  </property>
  <widget class="QLineEdit" name="multiSub1xPercentEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Enter whole numbers 1 - 100</string>
   </property>
   <property name="placeholderText">
    <string>Enter % to Give (1-100)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="multiSub1xAddressEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter Address to Sub1x to</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>451</width>
     <height>121</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>MultiSub1x allows you to automatically sub1x up to 100% of your stake or masternode reward to a list of other SUB1X addresses after it matures.
To Add: enter percentage to give and SUB1X address to add to the MultiSub1x vector.
To Delete: Enter address to delete and press delete.
MultiSub1x will not be activated unless you have clicked Activate</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="message">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>351</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>120</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="statusTip">
    <string>Add to MultiSub1x Vector</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="disableButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>120</width>
     <height>23</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Deactivate MultiSub1x</string>
   </property>
   <property name="text">
    <string>Deactivate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addressBookButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>160</y>
     <width>28</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Choose an address from the address book</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../sub1x.qrc">
     <normaloff>:/icons/address-book</normaloff>:/icons/address-book</iconset>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Percentage of stake to sub1x</string>
   </property>
   <property name="text">
    <string>Percentage:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Address to sub1x portion of stake to</string>
   </property>
   <property name="text">
    <string>Address:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>120</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="statusTip">
    <string>Delete Address From MultiSub1x Vector</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="activateButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>120</width>
     <height>23</height>
    </rect>
   </property>
   <property name="statusTip">
    <string>Activate MultiSub1x</string>
   </property>
   <property name="text">
    <string>Activate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>251</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="statusTip">
    <string>View MultiSub1x Vector</string>
   </property>
   <property name="text">
    <string>View MultiSub1x</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>351</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="multiSub1xStakeCheckBox">
      <property name="text">
       <string>Sub1x For Stakes</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="multiSub1xMasternodeCheckBox">
      <property name="text">
       <string>Sub1x For Masternode Rewards</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../sub1x.qrc"/>
 </resources>
 <connections/>
</ui>
